ocs_ci/deployment/terraform.py

```diff
@@ -121,3 +121,24 @@ def destroy_module(self, tfvars, module):
         logger.info(f"Destroying the module: {module}")
         cmd = f"terraform destroy -auto-approve -target={module} '-var-file={tfvars}' '{self.path}'"
         run_cmd(cmd, timeout=1200)
+
+    def change_statefile(self, module, resource_type, resource_name, instance):
+        """
+        Remove the records from the state file so terraform will no longer be
+        tracking the corresponding remote objects.
+
+        Args:
+            tfstate (str): path to terraform.tfstate file
```

Comment:
This param is not in use in the function.

Reply:
done